packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -221,6 +221,9 @@ class SystemChrome {
   /// The `orientation` argument is a list of [DeviceOrientation] enum values.
   /// The empty list causes the application to defer to the operating system
   /// default.
+  ///
+  /// **Important:** This setting will only be respected on iPad if multitasking is disabled for your app.
+  /// See: https://github.com/flutter/flutter/issues/27235
```

Comment:
This should explain exactly when this does and when it doesn't work instead of linking to an issue.